guide/installation-and-configuration/version-control.markdown

```diff
@@ -46,9 +70,24 @@ potentially bad merges. This assumes that no development takes place in
       chdir => "/var/cfengine/masterfiles";
     }
 ```
+This policy will then regularly and periodically pulling the masterfiles
```

Comment:
Minor detail: I vaguely recall that there is a bug which either misrenders or fails to render the page correctly if there is no blank line between the text and example. I might be misremembering though. You don't need to fix it now, but keep an eye on the documentation after this is merged.